.gitignore

```diff
@@ -9,3 +9,11 @@
 
 # rspec failure tracking
 .rspec_status
+
+# rails support app files
+/spec/support/rails_app/.bundle
+/spec/support/rails_app/log/*
+/spec/support/rails_app/tmp/*
+/spec/support/rails_app!/log/.keep
```

Comment:
I think you want to ignore the full dir, not just the `.keep` files, right? Those files are just there so that the dir shows up in git.

Reply:
Potentially?  I just copied this direct from what is auto setup using `rails new <app-name> ...`

Reply:
Interesting... Well if it was auto-generated, probably best not to mess with it 🤷‍♂